src/components/ui/sliders/productSlider/swiper/navigation.scss

```diff
@@ -0,0 +1,57 @@
+.swiper-button-prev,
+.swiper-button-next {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  max-height: 400px;
+  height: 100%;
+  width: 32px;
+  border: 1px solid #b4bdc4;
```

Comment:
the same situation with the:
1. scss vars
2. comments
4. already-made typography and vars

Reply:
4. important